wp-cli/vip-filesystem.php

```diff
@@ -77,14 +96,21 @@ public function update_filesizes( $args, $assoc_args ) {
 			WP_CLI::halt( 1 );
 		}
 
-		$this->progress = \WP_CLI\Utils\make_progress_bar(
-			'Checking ' . number_format( $attachment_count ) . ' attachments', $attachment_count );
+		WP_CLI::confirm( sprintf( 'Should we start processing %s attachments?', number_format( $attachment_count ) ), $assoc_args );
+
+		$max_id = $wpdb->get_var( 'SELECT ID FROM ' . $wpdb->posts . ' ORDER BY ID DESC LIMIT 1' );
+
+		$end_index = $start_index + $batch_size;
```

Comment:
Are we at all concerned about how accurate the `$end_index` would be? It is very likely that the post ID will not be sequential

Reply:
With the way it's written, we just loop through all post IDs until we hit the highest one. We will probably have batches where there are 0 results or 0 attachments but that's okay.